spec/.straight/config.yml

```diff
@@ -45,6 +45,8 @@ blockchain_adapters:
 
 insight_url: "https://insight.mycelium.com/api"
 insight_test_url: "https://test-insight.bitpay.com"
+
+insight_twebsocket_url: "wss://insight.mycelium.com"
```

Comment:
What's `insight_twebsocket_url`? I mean, why `twebsocket` and not just `socket`? If there is a reason and it's not a typo, an explanatory comment would be helpful.